sources/packages/backend/apps/api/src/route-controllers/education-program/education-program.controller.service.ts

```diff
@@ -90,6 +91,7 @@ export class EducationProgramControllerService {
       "Not able to a save the program due to an invalid request or " +
       "SABC code is duplicated.",
   })
+  @PayloadMaxSize(20480)
```

Comment:
Correct me if wrong, we can use this in controller instead of controller.service right?

Reply:
I am under the impression that they would not work if not in the controller, please correct me if I am wrong. Either way, the guards are to be added to the controllers, unless I am missing something.

Reply:
Yes, you're right. It should be only in the controllers. Thanks!